### PR TITLE
Renaming CI legs

### PR DESCRIPTION
**Pull request by Anipik**
The previous pr was alwayus using the mac machines with missing dependencies so just creating a new pr to avoid those failures.

@eerhardt is it fine to run centos just with netcoreapp3.0 and ubuntu with netcoreapp2.1 ?

### CODE REVIEW

**Comment by Anipik** [comment]
@artidoro the mac ci legs are still failing

**Comment by artidoro** [comment]
Looks a lot better already!

I have the following comments:
- Could you Pascal case the names? Netcoreapp -> NetCoreApp
- Could you remove the app from the name? NetCoreApp -> NetCore
- Could you do the same for netfx? netfx -> NetFx
- Could you specify which version of NetFx we are using? (if you think it's useful)

**Comment by artidoro** [comment]
On a separate note, is it possible to add a period in the name? 
For example NetCore_3.0 or NetCore3.0 instead of NetCore30.

**Comment by Anipik** [comment]
>  Could you remove the app from the name? NetCoreApp -> NetCore

Actually eric told me to keep that as we dont use netcore anywhere

> Could you do the same for netfx? netfx -> NetFx

Its actually azure thats changing it automatiically from uppercase to lowercase 

> Could you specify which version of NetFx we are using? (if you think it's useful)

Its not that useful but I can add that, as it may fix our lowercase problem

**Comment by artidoro** [comment]
It is strange that it is not keeping the Pascal case on NetCoreApp!

**Comment by Anipik** [comment]
> It is strange that it is not keeping the Pascal case on NetCoreApp!

It does weird things  :P

File: .vsts-dotnet-ci.yml [hunk 1]
```diff
@@ -13,71 +13,69 @@ resources:
...
-      Build_Debug_Intrinsics:
+      Debug_Build:
         _configuration: Debug-Intrinsics
         _config_short: DI
         _includeBenchmarkData: false
-      Build_Release_Intrinsics:
+      Release_Build:
         _configuration: Release-Intrinsics
-        _config_short: RI
+        _config_short: R
```
Comment (danmosemsft) [inline .vsts-dotnet-ci.yml]: RI?

File: .vsts-dotnet-ci.yml [hunk 2]
```diff
@@ -13,71 +13,69 @@ resources:
...
+    buildScript: build.cmd
     queue:
-      name: Hosted macOS
+      name: Hosted VS2017
 
 - template: /build/ci/phase-template.yml
   parameters:
-    name: Windows_NetFx
+    name: Windows_x64_NetFx461
     buildScript: build.cmd
```
Comment (danmosemsft) [inline .vsts-dotnet-ci.yml]: explicit `architecture: x64` ?

**Comment by Anipik** [comment]
> Next time if you can avoid reordering, the diff is easier to read.

Actually ML team wanted these reorderings. but yeah i will split the reordering into a separate commit from next time

File: .vsts-dotnet-ci.yml (hunk 2 above)
Comment (Anipik) [reply .vsts-dotnet-ci.yml]: Actually the default value of this architecture variable is x64 which we set here https://github.com/dotnet/machinelearning/pull/3553/files/251954988b8ebd8e2dd8c1eb420a5e7c09095dac#diff-51ce39ab4212d73da272ba04e66b0b3bR3

**Comment by eerhardt** [comment]
Why do we need “Build” in all the names?

**Comment by Anipik** [comment]
> Why do we need “Build” in all the names?

chocsta mentioned in the previous PR and naming the job matrices as debug and release can cause some problems with azure dev ops https://github.com/dotnet/machinelearning/pull/3314#discussion_r275455062

**Comment by eerhardt** [comment]
> @eerhardt is it fine to run centos just with netcoreapp3.0 and ubuntu with netcoreapp2.1 ?

It was nice getting a mix of the two that we were doing previously, but I think we should be OK.

**Comment by shauheen** [comment]
@Anipik this seems to have broken the build badges on the main readme.

**Comment by Anipik** [comment]
@shauheen We need to  update the job names there as well. i Will put up the PR for it.

**Comment by Anipik** [comment]
We need to update the links

**Comment by Anipik** [comment]
i found the new links to apis

**Pull request merged**